source/Ship.cpp

```diff
@@ -75,18 +75,6 @@ namespace {
 	}
 }
 
-const vector<string> Ship::CATEGORIES = {
-	"Transport",
-	"Light Freighter",
-	"Heavy Freighter",
-	"Interceptor",
-	"Light Warship",
-	"Medium Warship",
-	"Heavy Warship",
-	"Fighter",
-	"Drone"
-};
-
 
 
```

Comment:
One line between the end of namespace and the start of class methods is fine (as it was when categories were here).